protostar/commands/build/build_command.py

```diff
@@ -53,11 +52,13 @@ async def run(self, args):
             log_color_provider.colorize("GRAY", "Building projects' contracts")
         ):
             try:
-                self._project_compiler.compile(
-                    output_dir=args.output,
-                    relative_cairo_path=args.cairo_path,
-                    disable_hint_validation=args.disable_hint_validation,
+                self._project_compiler.set_config(
```

Comment:
Could this mutability be avoided? It suggests that the config can be changed during compilation, which I guess would result in crashes.

Reply:
I'll make it a parameter of `compile*` functions instead.